strongbox-commons/src/main/resources/application.yaml

```diff
@@ -33,6 +33,7 @@ strongbox:
   server:
     database:
       path: ${strongbox.vault}/db
+# Changes to this proeprties must also be applied to [strongbox/strongbox-web-integration-tests] and possibly the pipeline. Please ping us in the main channel to confirm.
```

Comment:
@sbespalov shouldn't this be under the `setenv.conf` and `strongbox-distribution/pom.xml`? I was thinking we should add this comment at the places where the environment variables are actually being set/exposed.

Reply:
@steve-todorov changes on `setenv.conf` and `strongbox-distribution/pom.xml` will not affect on anything :)
only this properties changes will affect on env variables

Reply:
Thanks for clarifying! :)